StandardConfig/production/MarlinStdReco.xml

```diff
@@ -45,8 +45,8 @@
     <!-- ***** Input files constants ***** -->    
     <!-- Special Beamcal overlay background file -->
     <constant name="BeamCalBackgroundFile" value="HighLevelReco/BeamCalBackground/BeamCal_bg_E500-TDR_ws.root" />
-    <!-- Particle identification PDF root file -->
-    <constant name="PidPDFFile" value="HighLevelReco/PIDFiles/LikelihoodPID_Histogram_v01.root" />
+    <!-- Particle identification PDF root file for l5 detector -->
+    <constant name="PidPDFFile" value="HighLevelReco/PIDFiles/LikelihoodPID_Standard_l5_v01.root" />
```

Comment:
What is v01 here ? Should be v02. Is it a typo or did you really use the model v01 ?

Reply:
I added new PID template file in HighLevelReco/PIDTools
File name is OK.

Reply:
v01 does not mean detector model. 
It just means template file tag.